Example/StripeConnectExample/StripeConnectExample/AppStartViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ViewController.swift
+//  StripeConnectExample
+//
+//  Created by Chris Mays on 8/21/24.
+//
+
+@_spi(PrivateBetaConnect) import StripeConnect
+import StripeCore
+import UIKit
+import SwiftUI
+
+struct AppLoadingView: View {
+    @Environment(\.viewControllerPresenter) var viewControllerPresenter
+    @State var loadingState: LoadingState = .loading
+    @State var displayLoadTimeReassurance: Bool = false
+    @State var loadTimeReassuranceWorkItem: DispatchWorkItem?
+
+    enum LoadingState {
+        case loading
+        case failed(reason: String)
+        case finished
+    }
+    
+    var body: some View {
+        VStack(spacing: 10) {
+            switch loadingState {
+            case .loading:
+                ProgressView()
+                Text("Warming up the server...This might take several seconds")
+                    .opacity(displayLoadTimeReassurance ? 1.0 : 0.0)
+                    .multilineTextAlignment(.center)
```

Comment:
Nice touch!

Reply:
supernit:
- Use ellipsis instead of three dots
- Adds space

```suggestion
                Text("Warming up the server… This might take several seconds")
                    .opacity(displayLoadTimeReassurance ? 1.0 : 0.0)
                    .multilineTextAlignment(.center)
```